metadata-ingestion/src/datahub/telemetry/telemetry.py

```diff
@@ -274,6 +275,13 @@ def capture_exception(self, e: BaseException) -> None:
             logger.warning("Failed to capture exception in Sentry.", exc_info=e)
 
     def init_tracking(self) -> None:
+        try:
+            graph = get_default_graph()
+            server_config = graph.test_connection()
```

Comment:
I don't feel great about this, since it means that _ALL_ CLI invocations will try to connect to GMS, even when they don't actually need to

given that it now persists the server telemetry state, I think it's fine to not do this